.changelog/118.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
This seems like more of a breaking change than an improvement...

Reply:
@briancain I updated the changelog to be a breaking change release note.